pkg/antctl/raw/supportbundle/command.go

```diff
@@ -734,6 +751,16 @@ func downloadPodLogs(k8sClient kubernetes.Interface, comp string, namespace stri
 		errors = append(errors, err)
 	}
 
+	gzFileName := filepath.Join(dir, comp+".tar.gz")
+
+	f, err := os.Create(gzFileName)
+	if err != nil {
+		errors = append(errors, err)
+	} else {
+		_, err := compress.PackDir(tmpDir, f)
+		errors = append(errors, err)
```

Comment:
if err in 60 is nil, it'll append a nil to errors. Maybe skip appending if err is nil.

Reply:
the error aggregator interface actually will filter out the nil error in the final output phase. So it's safe to append nil error to it. Although it's a little confusing for the users.